plugins/modules/idrac_license.py

```diff
@@ -0,0 +1,825 @@
+      file_name: "license_file"
+      ip_address: "192.168.0.1"
+      username: "username"
+      password: "password"
+      ignore_certificate_warning: "on"
+
+- name: Import a license to iDRAC from local
+  dellemc.openmanage.idrac_license:
+    idrac_ip: 198.162.0.1
+    idrac_user: "username"
+    idrac_password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    import: true
+    share_parameters:
+      file_name: "license_file_name.xml"
+      share_type: local
+      share_name: "/path/to/share"
+
+- name: Import a license to iDRAC from NFS share
+  dellemc.openmanage.idrac_license:
+    idrac_ip: 198.162.0.1
+    idrac_user: "username"
+    idrac_password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    import: true
+    share_parameters:
+      file_name: "license_file_name.xml"
+      share_type: nfs
+      ip_address: "192.168.0.1"
+      share_name: "/path/to/share"
+
+- name: Import a license to iDRAC from CIFS share
+  dellemc.openmanage.idrac_license:
+    idrac_ip: 198.162.0.1
+    idrac_user: "username"
+    idrac_password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    import: true
+    share_parameters:
+      file_name: "license_file_name.xml"
+      share_type: cifs
+      ip_address: "192.168.0.1"
+      share_name: "/path/to/share"
+      username: "username"
+      password: "password"
+
+- name: Import a license to iDRAC from HTTP share
+  dellemc.openmanage.idrac_license:
+    idrac_ip: 198.162.0.1
+    idrac_user: "username"
+    idrac_password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    import: true
+    share_parameters:
+      file_name: "license_file_name.xml"
+      share_type: http
+      ip_address: "192.168.0.1"
+      share_name: "/path/to/share"
+      username: "username"
+      password: "password"
+
+- name: Import a license to iDRAC from HTTPS share via proxy
+  dellemc.openmanage.idrac_license:
+    idrac_ip: 198.162.0.1
+    idrac_user: "username"
+    idrac_password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    import: true
+    share_parameters:
+      file_name: "license_file_name.xml"
+      share_type: https
+      ip_address: "192.168.0.1"
+      share_name: "/path/to/share"
+      username: "username"
+      password: "password"
+      proxy_support: "parameters_proxy"
+      proxy_server: "192.168.0.2"
+      proxy_port: "808"
+      proxy_username: "proxy_username"
+      proxy_password: "proxy_password"
+
+- name: Delete a License from iDRAC
+  dellemc.openmanage.idrac_license:
+    idrac_ip: 198.162.0.1
+    idrac_user: "username"
+    idrac_password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+    license_id: "LICENCE_123"
+    delete: true
+"""
+
+RETURN = r'''
+---
+msg:
+  type: str
+  description: Status of the license operation.
+  returned: always
+  sample: "Successfully exported the license."
+job_details:
+    description: Returns the output for status of the job.
+    returned: For import and export operations
```

Comment:
can we add either in notes or elsewhere regarding job details not returned if share_type is local

Reply:
Done